c/crossmemory.c

```diff
@@ -71,13 +71,88 @@ const CrossMemoryServerName CMS_DEFAULT_SERVER_NAME = {CROSS_MEMORY_DEFAULT_SERV
+  [RC_CMS_GLOBAL_AREA_NULL] = "Global area address is NULL",
+  [RC_CMS_GLOBAL_AREA_BAD_EYECATCHER] = "Global area has bad eyecatcher",
+  [RC_CMS_SERVER_NULL] = "Server address is NULL",
+  [RC_CMS_FUNCTION_NULL] = "Service function address is NULL",
+  [RC_CMS_AXSET_FAILED] = "PC-ss not set, AXSET failed",
+  [RC_CMS_LXRES_FAILED] = "PC-ss/cp not set, LXRES failed",
+  [RC_CMS_ETCRE_FAILED] = "PC-ss/cp not set, build entry table failed",
+  [RC_CMS_ETCON_FAILED] = "PC-ss/cp not set, ETCON failed",
+  [RC_CMS_DUPLICATE_SERVER] = "A duplicate server is running",
+  [RC_CMS_ISGENQ_FAILED] = "Server not locked",
+  [RC_CMS_ECSA_ALLOC_FAILED] = "Resource manager ECSA storage not allocated",
+  [RC_CMS_NAME_TOKEN_CREATE_FAILED] = "Name/Token create failed",
+  [RC_CMS_SERVER_NOT_AVAILABLE] = "Server not reachable",
+  [RC_CMS_NAME_TOKEN_NOT_FOUND] = "Name/Token not found",
+  [RC_CMS_NAME_TOKEN_BAD_EYECATCHER] = "Name/Token has bad eyecatcher",
+  [RC_CMS_LPA_ADD_FAILED] = "Module not loaded into LPA",
+  [RC_CMS_LPA_DELETE_FAILED] = "Module not deleted from LPA",
   [RC_CMS_SERVER_NOT_READY] = "Server is not running",
+  [RC_CMS_NAME_TOKEN_DELETE_FAILED] = "Name/Token delete failed", 
+  [RC_CMS_RACF_ROUTE_LIST_FAILED] = "RACF route list failed ",
   [RC_CMS_PERMISSION_DENIED] = "Permission denied",
-  [RC_CMS_SERVER_ABENDED] = "Cross-memory call ABENDed",
+  [RC_CMS_PC_ENV_NOT_ESTABLISHED] = "PC environment not established",
+  [RC_CMS_PC_ENV_NOT_TERMINATED] = "PC environment not terminated",
+  [RC_CMS_PC_SERVICE_ABEND_DETECTED] = "PC service abended",
+  [RC_CMS_PC_RECOVERY_ENV_FAILED] = "PC recovery environment failed",
+  [RC_CMS_SERVER_ABENDED] = "Cross-memory call abended",
+  [RC_CMS_VSNPRINTF_FAILED] = "vsnprintf failed",
+  [RC_CMS_MESSAGE_TOO_LONG] = "Message too long",
+  [RC_CMS_LOGGING_CNTX_NOT_FOUND] = "Logging context not found",
+  [RC_CMS_RECOVERY_CNTX_NOT_FOUND] = "Recovery context not found",
+  [RC_CMS_SERVICE_NOT_INITIALIZED] = "Service is not initialized",
+  [RC_CMS_ZVT_NULL] = "ZVT is NULL",
+  [RC_CMS_ZVTE_NULL] = "ZVTE is NULL",
+  [RC_CMS_ZVTE_NOT_ALLOCATED] = "ZVTE is not allocated",
+  [RC_CMS_SERVICE_NOT_RELOCATABLE] = "Service function out of range, service not relocated",
+  [RC_CMS_MAIN_LOOP_FAILED] = "Main loop unexpectedly terminated",
+  [RC_CMS_IMPROPER_SERVICE_AS] = "Invalid service address space",
+  [RC_CMS_BAD_SERVER_KEY] = "Core server started with wrong key", 
+  [RC_CMS_RESMGR_NOT_ADDED] = "RESMGR not added",
+  [RC_CMS_RESMGR_NOT_REMOVED] = "RESMGR not removed",
+  [RC_CMS_RESMGR_NOT_LOCKED] = "RESMGR not locked",
+  [RC_CMS_RESMGR_NAMETOKEN_FAILED] = "RESMGR Name/Token not retrieved",
   [RC_CMS_WRONG_SERVER_VERSION] = "Wrong server version",
+  [RC_CMS_LATENT_PARM_NULL] = "Latent parameter is NULL",
+  [RC_CMS_USER_PARM_NULL] = "User parameter is NULL",
+  [RC_CMS_PC_HDLR_PARM_BAD_EYECATCHER] = "PC handler parameter has invalid eyecatcher",
+  [RC_CMS_ZERO_PC_NUMBER] = "Zero PC number",
```

Comment:
`PC is unavailable`

I know this does not exactly match the name but that's usually what that means. It's zeroed out when someone cancels ZIS.